.github/workflows/ci.yml

```diff
@@ -79,15 +79,3 @@ jobs:
         ANY_MODIFIED: ${{ steps.changed-files.outputs.any_modified }}
       shell: bash
       run: make test DOCKER_ORG="ghcr.io/bufbuild"
-    - name: Push
-      if: github.repository == 'bufbuild/plugins'
-      shell: bash
-      env:
-        ALL_MODIFIED_FILES: ${{ steps.changed-files.outputs.all_modified_files }}
-        ANY_MODIFIED: ${{ steps.changed-files.outputs.any_modified }}
-        BUF_ALPHA_SUPPRESS_WARNINGS: 1
-        BSR_USER: ${{ secrets.BSR_USER }}
-        BSR_TOKEN: ${{ secrets.BSR_TOKEN }}
```

Comment:
Let's create an internal issue to track removing this token from the repository and the BSR.

Keep this for now until we confirm everything is groovey.